content/api_chef_server.md

```diff
@@ -2604,6 +2606,184 @@ The response is similar to:
 </tbody>
 </table>
 
+### POST
+
+The `POST` method is used to create a container.
+
```

Comment:
Add here something like `NOTE: while this method is implemented, it is not supported, and use will likely break your system, or not have the effect you expect.`